Gaijinizer patch/Map047.txt

```diff
@@ -255,129 +259,135 @@ What………!!What………! !#MTLed
 
 まだお外で遊びたいのね…。
 > CONTEXT: Map047/events/38/pages/0/189/Dialogue/enemy4
-………so. I still want to play outside ....#MTLed
+... I see.
+
+You still want to play outside?#Ron
 > END STRING
 
 > BEGIN STRING
 でも、ママもあきらめないわよ。
 
 あなたとおうちに帰るまで、一緒に遊んであげる。
 > CONTEXT: Map047/events/38/pages/0/193/Dialogue/enemy4
-But my mom won't give up. I will play with you
-until I go home with you.#MTLed
+But mommy's not giving up on you. I'm going to
+play with you until we go back home together.#Ron
 > END STRING
 
 > BEGIN STRING
 でも、くれぐれも他のこわい悪魔に気をつけてね…
 
 危なくないようにするのよ…？
 > CONTEXT: Map047/events/38/pages/0/197/Dialogue/enemy4
-But watch out for other scary demons ...
-Do not be dangerous ...?#MTLed
+Watch out for other scary demons... Don't do
+anything dangerous, okay?#Ron
 > END STRING
 
 > BEGIN STRING
 はあっ…！はあっ…！
 
 と、とりあえずは……
 > CONTEXT: Map047/events/38/pages/0/220/Dialogue/kazuya-0
-What…! What…! Well, for now ...#MTLed
+Haaha...! Haaah...!
+That should do it, for now...#Ron
 > END STRING
 
 > BEGIN STRING
 大丈夫か！？\\N[1]！！
 
 どこか負傷したのか！？
 > CONTEXT: Map047/events/38/pages/0/229/Dialogue/yuuma
-Are you okay! ? \\N[1]!!I was injured
-somewhere! ?#MTLed
+\\N[1]!! Are you okay!?
+
+Do you have any injuries!?#Ron
 > END STRING
 
 > BEGIN STRING
 わ、わからない……
 
 全身がピリピリ痛むんだ…、視界が…少しぼやけてる…。
 > CONTEXT: Map047/events/38/pages/0/233/Dialogue/kazuya-2
-Wow, I don't know ...... My whole body is
-throbbing pain...it's a bit blurry.#MTLed
+I, I don't know...... My whole body's tingling...
+my vision's a little... blurry.#Ron
 > END STRING
 
 > BEGIN STRING
 でも………な、なんだかそれが気持ちよくて……。
 > CONTEXT: Map047/events/38/pages/0/237/Dialogue/kazuya-0
-But...... what, it feels good......#MTLed
+But it kinda... kinda feels good...#Ron
 > END STRING
 
 > BEGIN STRING
 あーあ、それ、マザーラビットの母乳の症状だよ。
 
 \\N[1]、最初に口に含んだときに少し飲んだね？
 > CONTEXT: Map047/events/38/pages/0/241/Dialogue/hyakki
-Ah, ah, that's the mother's breast milk symptoms.
-\\N[1], you drank a little when you first got it
-in your mouth, right?#MTLed
+Aaaah, why that, that's an effect from Mother
+Rabbit's milk. \\N[1], you must have drunk a
+little when her breast was in your mouth.#Ron
 > END STRING
 
 > BEGIN STRING
 ひ、百鬼……！
 > CONTEXT: Map047/events/38/pages/0/251/Dialogue/kazuya
-Hi, Hyakki ......!#MTLed
+Hy-, Hyakki ......!#Ron
 > END STRING
 
 > BEGIN STRING
 腕、見てみなよ。
 
 そろそろ目に見えて変化が出始めてるだろうからサ。
 > CONTEXT: Map047/events/38/pages/0/253/Dialogue/hyakki
-Arms, look! Because it will be visible and change
-will start to come out.#MTLed
+Your arms, take a look at 'em! 'cause soon enough
+there's gonna be some visible changes starting.#Ron
 > END STRING
 
 > BEGIN STRING
 腕を見てみると、白いふわふわした産毛のような……
 
 しかし明らかに人間のものでない体毛が生え始めていた！
 > CONTEXT: Map047/events/38/pages/0/258/Dialogue
-Looking at the arm, it looks like a white lightly downy
-hair...But the hair that was obviously not human was
-beginning to grow!#MTLed
+Looking at my arms, there's white fluffy hair...
```

Comment:
I feel this can be phrased better, just making a note to myself to examine it later and edit this....